HabitTracker.Rutwik2005/HabitTracker.Rutwik2005/Program.cs

```diff
@@ -0,0 +1,230 @@
+﻿using Microsoft.Data.Sqlite;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+class Program
+{
+    public static string connectionString = @"Data Source=HabitTracker.Rutwik2005.db";
+    public static void Main(string[] args)
+    {
+
+        using (var connection = new SqliteConnection(connectionString))
```

Comment:
I will only comment on it here, but this is throughout your code.
Remember that you can use the simpler "Using Declaration" syntax to save an indentation level against the "Using Statement" syntax:

❌ 
```
// Using Statement
using (var connection = new SqliteConnection(connectionString))
{
    // ...
}
```

✔️ 
```
// Using Declaration
using var connection = new SqliteConnection(connectionString);
// ...
```